sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -58,15 +67,102 @@ def list_repositories(self, **kwargs):
         :keyword last: Query parameter for the last item in the previous call. Ensuing
             call will return values after last lexically
         :type last: str
-        :keyword results_per_page: Numer of repositories to return in a single page
+        :keyword results_per_page: Numer of repositories to return per page
```

Comment:
nit
edit: nit for everywhere you copy/pasted this 😄 

```suggestion
        :keyword results_per_page: Number of repositories to return per page
```

Reply:
My middle school spelling abilities strike again!